Description: disable test because it depends on postject, which is dfsg-excluded
 HELP is welcome to solve this
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2023-11-30
Forwarded: not-needed
--- a/test/parallel/parallel.status
+++ b/test/parallel/parallel.status
@@ -16,6 +16,7 @@
 test-dns                             : SKIP
 test-net-connect-immediate-finish    : SKIP
 test-net-better-error-messages-port-hostname : SKIP
+test-single-executable-application: SKIP
 
 # in debian build env, skip because it somehow tries to access network
 test-https-connect-address-family : SKIP 
